app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -332,6 +332,10 @@ class BrowserFragment :
             owner = this,
             view = rootView
         )
+        if (requireComponents.appStore.state.autoplayRulesChanged) {
```

Comment:
Would've been nice to observe the store and do a direct mapping of the state instead of treating the AppStore as just a singleton containing a value but we can refactor later to this flow and with Compose it will just work.
Tested it and solves the issue. Thanks!